SECURITY.md

```diff
@@ -0,0 +1,78 @@
+Matt Butcher       | [link](https://keybase.io/technosophos/pgp_keys.asc) | ABA2 5295 98F6 626C 420D 335B 62F4 9E74 7D91 1B60
+Matt Farina        | [link](https://keybase.io/mattfarina/pgp_keys.asc)   | 672C 657B E06B 4B30 969C 4A57 4614 49C2 5E36 B98E
+Matt Fisher        | [link](https://keybase.io/bacongobbler/pgp_keys.asc) | 967F 8AC5 E221 6F9F 4FD2 70AD 92AA 783C BAAE 8E3B
+Michelle Noorali   ||
+Nikhil Manchanda   | [link](https://keybase.io/slicknik/pgp_keys.asc)     | 5674 7211 71F8 CD59 F285 353D 9692 757E BCD3 D66C
+Vic Iglesias       | [link](https://keybase.io/vicnastea/pgp_keys.asc)    | 43A7 994E F6C7 7D7F 147B C152 381B ABCF F47D EAFC
+
+### When To Send A Report
+
+You think you have found a vulnerability in a Helm project or a dependency of a Helm project. This can be any of the repositories on the [Helm GitHub organization](https://github.com/helm).
+
+### When Not To Send A Report
+
+* If a vulnerability has been found in an application deployed by a Helm Chart. Instead, contact the application maintainers
+* For guidance on securing Helm. Please see the documentation or ask in one of the support channels
+* You are looking for Helm applying security updates
+
+### Security Vulnerability Response
+
+Each report will be reviewed and receipt acknowledged within 3 business days. This will set off the security review process detailed below.
+
+Any vulnerability information shared with the security team stays within the Helm project and will not be shared with others unless it is necessary to fix the issue. Information is shared only on a need to know basis.
+
+As the security issue through triage, identification, and release the reporter of the security vulnerability will be notified. Additional questions about the vulnerability may also be asked of the reporter.
+
+### Public Disclosure
+
+A public disclosure of security vulnerabilities is released alongside release updates or details that fix the vulnerability. We try to fully disclose vulnerabilities once a mitigation strategy is available. Our goal is to perform a release and public disclosure quickly and in a timetable that works well for users. For example, a release may be ready on a Friday but for the sake of users may be delayed to a Monday.
+
+CVEs will be assigned to vulnerabilities. Due to the process and time it takes to obtain a CVE ID disclosures will happen first. Once the disclosure is public the process will begin to obtain a CVE ID. Once the ID has been assigned the disclosure will be updated.
+
+If the vulnerability reporter would like their name and details shared as part of the disclosure process we are happy to. We will ask permission and for the way the reporter would like to be identified. We appreciate vulnerability reports and would like to credit reporters if they would like the credit.
+
+## Security Team Membership
+
+The security team is made up of Helm project maintainers who are willing and able to respond to vulnerability reports.
```

Comment:
I think me might say "is made up of a subset of the Helm project maintainers" -- and I suggest this because I worry that perhaps people who are not on the list could get mistakenly contacted directly.

Reply:
Good call. Better to be explicit than implicit here.